docs/secondary-navigation.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: sub-navigation
+order: 5
+title: Secondary navigation
+description: Allow users to navigate between pages scoped to a particular section.
+---
+
+{% from "example/macro.njk" import appExample %}
+
+{{ appExample({
+  component: "secondary-navigation"
+}) }}
+
+## When to use this component
+
+Use the secondary navigation when you already have a [primary navigation](primary-navigation) and need to let users navigate between pages that are scoped to a particular section or concept.
+
+For example, if your service is a case management system, you might use the secondary navigation if there are multiple pages associated with a single case.
+
+{{ appExample({
+  component: "secondary-navigation",
+  example: "with-header"
+}) }}
```

Comment:
@paulrobertlloyd  I added this second example as I thought for this component it was helpful to see the component within a bit more context (ie with a heading above and below it) to help distinguish it from the primary nav.  But that seemed too much for the preview at the very top of the page...

Reply:
Makes sense to me.